processor/ratelimitprocessor/gubernator.go

```diff
@@ -127,7 +127,7 @@ func newGubernatorRateLimiter(cfg *Config, logger *zap.Logger, telemetryBuilder
 	return &gubernatorRateLimiter{
 		cfg:                cfg,
 		logger:             logger,
-		behavior:           gubernator.Behavior_BATCHING,
+		behavior:           cfg.GubernatorBehavior,
```

Comment:
You don't need this anymore, you can just use `cfg.GubernatorBehavior` since we are storing the whole config above